interpreter/text/parser.mly

```diff
@@ -583,25 +584,28 @@ active_elemref_list :
       fun c lookup -> List.map f ($1 c lookup) }
 
 elem :
-  | LPAR ELEM bind_var_opt PASSIVE passive_elemref_list RPAR
+  | LPAR ELEM bind_var_opt PASSIVE elem_type passive_elemref_list RPAR
```

Comment:
Hm, so with the elem_type being mandatory, it seems like we could drop the extra "passive" keyword?

Reply:
Yes, I think you're right.